pulsar-client-admin-shaded/pom.xml

```diff
@@ -131,6 +131,7 @@
                   <include>com.yahoo.datasketches:sketches-core</include>
                   <include>org.glassfish.jersey*:*</include>
                   <include>javax.ws.rs:*</include>
+                  <include>javax.xml.bind:*</include>
```

Comment:
@nodece this inclusion/relocation looks good. I think we need to do the same thing to [pulsar-client-all/pom.xml](https://github.com/apache/pulsar/blob/091ee2504ffbe6ec98e354b76e7f4c045e1914aa/pulsar-client-all/pom.xml#L184).

Reply:
I'll make a new PR to do that.

Reply:
@onobc Done.